inspire_json_merger/pre_filters.py

```diff
@@ -188,5 +188,28 @@ def _remove_if_falsy(pmap, key):
         return pmap
 
 
+def remove_duplicated_titles(root, head, update):
+    new_elements = []
+    for data in (root, head, update):
+        titles = data.get('titles')
+        if not titles:
```

Comment:
```suggestion
        if "titles" not in data:
```
as the schema guarantees it's never empty.

Reply:
It might be empty when root is empty (and we are cleaning it sometimes)